### ISSUE TITLE
Can STM32F411CEU6 be used instead of STM32F411CCU6TR

### ISSUE THREAD

**Issue by farcouet** [opiopan/simhid-g1000#3]
Hello
As the STM32F411CCU6TR is backorder everywhere I was trying to find a replacement. None are stock but there are dev board on ebay with a STM32F411CEU6 and I was thinking of ordering one and take it from that board.
The only difference that I am seeing is the flash that is double on the STM32F411CEU6. Can it be used? I am not familiar with those STMicro mcu.
Thanks,
Fred

**Comment by farcouet** [comment]
Hello!
Yes kind of. Situation was not that bad a year ago with Mouser and Digikey even if we started to hear about it. But since 9 months now it hits pretty bad! 
So perfect I have ordered the little dev board on ebay and will salvage the MCU from it! I have sent the PCB to be made by JLCPCB and kind of nice to have them solder the small 0402 instead of spending time on this! I was lucky as all other parts except the MCU and the USB connectors were available. I will salvage also the USB from the dev board!
I got the last port expander they had.
Still printing the parts and need to find a place for the laser cut & engraving.
Hopefully will complete this in few weeks (time to get also the board).
Thanks!
Fred
